rfc/2022/2022-06-13 - Engineering Decisions will be simplified to only include RFCs and ADRs.md

```diff
@@ -0,0 +1,74 @@
+* teams will more fully understand their problem at hand by writing (prose, not code!) about their challenge and their suggested solution
+
+**Platform crew leaders will benefit** from seeing more of the work being done in the open, and be able to steer decisions as needed without being directly involved. This will lead to higher-quality work on the Platform, and more efficient delivery of solutions, as the design stage will (in many cases) move faster.
+
+## Design
+
+* **Simplify the process**. The current process involves four artifacts that team members are required to understand and choose between: ADRs, Idea Docs, Design Docs, and RFCs. Practically, ADRs are the only one of these that are commonly requested. 
+Idea docs and design docs are removed from the process for simplicity. Teams of course may still generate idea docs and design docs. But if the work they are doing affects multiple teams, has architectural impacts, or there are alternatives that the Platform should consider, then an RFC should be created in addition to these docs.
+
+* **Author and track RFCs as github PRs**. RFC stands for Request for Comments, and the medium where an RFC is published should be optimized for discussion. PRs provide a framework for collecting suggestions, questions, and thoughts about code, until that code is in a state where everyoine involved agrees that it is ready for production. This same framework should be used for RFCs: the work plan (RFC text) is published, comments/suggestions are collected, answers to questions are incorporated into the text, then the RFC is “approved” via merge into the repo. Interested parties can easily “subscribe” to the RFC PR to get updates sent to their inbox.
+
+* **Create a friendly and descriptive RFC template** in github, that contains hints to get authors to quickly create RFCs with little friction (one already exists; update as needed). The goal is to enable an RFC author to get their idea into a sharable state in less than 60 minutes. Create a PR description template `.github/PULL_REQUEST_TEMPLATE/rfc.md` to ensure a standard way of describing the RFC outside of the RFC file body.
+
+* **Create a lightweight RFC PR process**. This could include setting a timebox around RFC lifetime, and a loose “approval” process that empowers the author to move forward once all comments have been resolved.
+
+* **Move RFCs to a more visible place**. They currently live in the folder `/platform/engineering/request-for-comment/` in the va.gov-team repo, move them to dir `/rfc` in the va.gov-platform-architecture repo.
+
+
+## Risks
+
+* Simplifying the process will not lead to the desired outcome of more work being done in public
+
+* The omission of design docs and idea docs will lead to less documentation
+
+* Using a PR process may introduce a barrier to commenting for those not familiar with Github's pull request system and process.
```

Comment:
Can you get those team members to provide examples of what their challenges are? Since we're already moving forward with github as the platform, we should identify what type of training aid we can put in place to assist them.

Reply:
We just had a thread recently where non-engineers were commenting on a RFC in slack instead of github